From 05aa4e9758b7afb0866081795e1d7c139861ac97 Mon Sep 17 00:00:00 2001
From: Julian Andres Klode <julian.klode@canonical.com>
Date: Thu, 8 Feb 2018 10:48:37 +0100
Subject: UBUNTU: Show only upstream version, hide rest in package_version
 variable

The complete package version can get a bit long, so only show the
upstream version in the menu and on the top of the console, and
hide the complete version in a package_version variable.

Bug-Ubuntu: https://bugs.launchpad.net/bugs/1723434
Last-Updated: 2018-02-08

Patch-Name: ubuntu-shorter-version-info.patch
---
 grub-core/normal/main.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/grub-core/normal/main.c b/grub-core/normal/main.c
index 0aa389fa1..d25a8212c 100644
--- a/grub-core/normal/main.c
+++ b/grub-core/normal/main.c
@@ -208,7 +208,7 @@ grub_normal_init_page (struct grub_term_output *term,
  
   grub_term_cls (term);
 
-  msg_formatted = grub_xasprintf (_("GNU GRUB  version %s"), PACKAGE_VERSION);
+  msg_formatted = grub_xasprintf (_("GNU GRUB  version %s"), VERSION);
   if (!msg_formatted)
     return;
  
@@ -561,6 +561,9 @@ GRUB_MOD_INIT(normal)
   grub_env_set ("grub_platform", GRUB_PLATFORM);
   grub_env_export ("grub_platform");
 
+  grub_env_set ("package_version", PACKAGE_VERSION);
+  grub_env_export ("package_version");
+
   grub_boot_time ("Normal module prepared");
 }
 
